### ISSUE TITLE
Combo box text unreadable on High Contrast White

### ISSUE THREAD

**Issue by huguesv** [Microsoft/PTVS#2606]
Using latest from master

![image](https://cloud.githubusercontent.com/assets/1696845/26598053/cfc57436-4528-11e7-85f1-878dd4de9296.png)

**Comment by huguesv** [comment]
In Environments tool window, it is partially correct.
![image](https://cloud.githubusercontent.com/assets/1696845/26598157/44ff125c-4529-11e7-9b7f-5d24a1bdd852.png)

**Comment by huguesv** [comment]
The hover colors should be white on dark purple on that last screenshot.

**Comment by huguesv** [comment]
Cookiecutter "Next" button is also unreadable.
![image](https://cloud.githubusercontent.com/assets/1696845/26598369/d85c2ed6-4529-11e7-9541-a47b92005446.png)

**Comment by huguesv** [comment]
That screenshot above where the hover color is light blue - that is fixed by restarting VS after changing windows theme.

**Comment by zooba** [comment]
We need to check, but I don't think high contrast white is required for us - fairly sure I saw somewhere that VS only supports high contrast dark. But I could be wrong?
